code/src/java/pcgen/io/exporttoken/EqToken.java

```diff
@@ -273,11 +257,11 @@ else if ("OR".equals(bString))
 						{
 							if (and_operation)
 							{
-								result = (result && eq.isType(bString));
+								result &= eq.isType(bString);
```

Comment:
Please do not change booleans to bitwise operations. &= applies a & b not a && b